### PR TITLE
fix space in expected log

### PR DESCRIPTION
**Pull request by casz**
https://github.com/jenkinsci/configuration-as-code-plugin/pull/1049

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
   <properties>
     <java.level>8</java.level>
     <jenkins.version>2.150.1</jenkins.version>
-    <jcasc.version>1.29</jcasc.version>
```
Comment (casz) [inline pom.xml]: perhaps best to let dependabot take care of the bump

**Comment by timja** [comment]
cc @oleg-nenashev

File: pom.xml (hunk 1 above)
Comment (timja) [reply pom.xml]: no point, the tests would fail

**Comment by casz** [comment]
shush @timja shush 🤣 I was about to write

@oleg-nenashev would appreciate a release with this fix as it helps resolve https://github.com/jenkinsci/bom/pull/78

**Pull request merged**